src/pages/Home.jsx

```diff
@@ -1,8 +1,53 @@
 import React from 'react'
+import dontStop from './assets/dontStop.jpg'
+import giveAll from './assets/giveAll.jpg'
+import dramaticImage from './assets/dramaticImage.JPG'
 
 const Home = () => {
   return (
-    <div>Home</div>
+    <div id="carouselExampleCaptions" class="container carousel slide carousel-fade" data-bs-ride="carousel">
```

Comment:
suggest to import Carousel from react-bootstrap to replace this carousel.

Also - change all `class` to `className`